extension/docs/WebComponentsIntegrationGuide.md

```diff
@@ -166,57 +164,9 @@ Next, use the Adyen `Component` to render the payment method, and collect the re
 
 If you haven't created the payment forms already in your frontend, follow the official Adyen [Web Components integration guide](https://docs.adyen.com/checkout/components-web#step-2-add-components).
 
-An `origin key` is a client-side key that is used to validate Adyen's JavaScript component library. It is required to render a Component.
-
-In order to get the origin key [update commercetools payment](https://docs.commercetools.com/http-api-projects-payments#update-payment) with `getOriginKeysRequest` custom field. 
-Note: if you want to get both origin keys and payment methods at once, you can set both `getOriginKeysRequest` and `getPaymentMethodsRequest` to the commercetools payment object within a single update request or on payment creation.
-
-An [update action](https://docs.commercetools.com/http-api-projects-payments#set-customfield) to set `getOriginKeysRequest` custom field.
-> Refer Adyen's [/originKeys](https://docs.adyen.com/api-explorer/#/PaymentSetupAndVerificationService/v52/post/originKeys) request to check all possible request payload parameters.
-
-```
-{
-  "version": "PAYMENT_VERSION",
-  "actions": [  
-    {
-      "action": "setCustomField",
-      "name": "getOriginKeysRequest",
-      "value": "{\"originDomains\":[\"https://www.your-company1.com\",\"https://www.your-company2.com\"]}"
-    }
-  ]
-}
-```
-
-The response contains a list of origin keys for all requested domains. For each list item, the key is the domain, and the value is its associated origin key. For more info see also [origin keys](https://docs.adyen.com/user-management/how-to-get-an-origin-key) 
-
-``` json
-{
- "originKeys":{
-    "https://www.your-company1.com":"pub.v2.99...",
-    "https://www.your-company2.com":"pub.v2.99...",
- }
-}
-```
-
-The commercetools payment representation example:
-
-``` json
-{
-  "custom": {
-    "type": {
-      "typeId": "type",
-      "key": "ctp-adyen-integration-web-components-payment-type"
-    },
-    "fields": {
-      "getOriginKeysRequest": "{\"originDomains\":[\"https://www.your-company1.com\",\"https://www.your-company2.com\"]}",
-      "getOriginKeysResponse": "{\"originKeys\":{\"https://www.your-company1.com\":\"pub.v2.99...\",\"https://www.your-company2.com\":\"pub.v2.99...\"}}"
-    }
-  }
-}
-```
+A `client key` is a client-side key that is used to validate Adyen's JavaScript component library. It is required to render a Component.
 
-Pass the `origin key` to your front end. Origin key is required to render an Adyen payment method Component.
- 
+In order to get a `client key`, follow the official [Adyen documentation](https://docs.adyen.com/development-resources/client-side-authentication#get-your-client-key).
```

Comment:
To find out how to generate a `clientKey`, follow the official [Adyen documentation](https://docs.adyen.com/development-resources/client-side-authentication)

Reply:
I didn't change this, I don't see much difference here.